modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -2,146 +2,88 @@
+        before_action :set_lighthouse_claim, :set_bgs_claim, :raise_if_no_claims_for_id!, :validate_id_with_icn!,
+                      only: %i[show]
 
-          render json: [] && return unless bgs_claims || lighthouse_claims
-          mapped_claims = map_claims(bgs_claims:, lighthouse_claims:)
-          blueprint_options = { base_url: request.base_url, veteran_id: params[:veteranId], view: :index, root: :data }
-          render json: ClaimsApi::V2::Blueprints::ClaimBlueprint.render(mapped_claims, blueprint_options)
+        def index
+          render json: claims_map
         end
 
         def show
-          lighthouse_claim = find_lighthouse_claim!(claim_id: params[:id])
-          benefit_claim_id = lighthouse_claim.present? ? lighthouse_claim.evss_id : params[:id]
-          bgs_claim = find_bgs_claim!(claim_id: benefit_claim_id)
-
-          if lighthouse_claim.blank? && bgs_claim.blank?
-            claims_v2_logging('claims_show', level: :warn, message: 'Claim not found.')
-            raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
-          end
-
-          validate_id_with_icn(bgs_claim, lighthouse_claim, params[:veteranId])
-
-          output = generate_show_output(bgs_claim:, lighthouse_claim:)
-          blueprint_options = { base_url: request.base_url, veteran_id: params[:veteranId], view: :show, root: :data }
-
-          render json: ClaimsApi::V2::Blueprints::ClaimBlueprint.render(output, blueprint_options)
+          render json: claim_map
         end
 
         private
 
-        def evss_docs_service
-          EVSS::DocumentsService.new(auth_headers)
+        def set_bgs_claims
+          @bgs_claims = find_bgs_claims!
         end
 
-        def bgs_phase_status_mapper
-          ClaimsApi::BGSClaimStatusMapper.new
+        def set_lighthouse_claims
+          @lighthouse_claims = ClaimsApi::AutoEstablishedClaim.where(veteran_icn: target_veteran.mpi.icn)
         end
 
-        def validate_id_with_icn(bgs_claim, lighthouse_claim, request_icn)
-          if bgs_claim&.dig(:benefit_claim_details_dto).present?
-            clm_prtcpnt_vet_id = bgs_claim&.dig(:benefit_claim_details_dto, :ptcpnt_vet_id)
-            clm_prtcpnt_clmnt_id = bgs_claim&.dig(:benefit_claim_details_dto, :ptcpnt_clmant_id)
-          end
-
-          veteran_icn = if lighthouse_claim.present? && lighthouse_claim['veteran_icn'].present?
-                          lighthouse_claim['veteran_icn']
-                        end
-
-          if clm_prtcpnt_cannot_access_claim?(clm_prtcpnt_vet_id, clm_prtcpnt_clmnt_id) && veteran_icn != request_icn
-            raise ::Common::Exceptions::ResourceNotFound.new(
-              detail: 'Invalid claim ID for the veteran identified.'
-            )
-          end
+        # Does not work exactly as it appears
+        def render_if_no_claims
+          render json: [] && return unless @bgs_claims || @lighthouse_claims
```

Comment:
@rockwellwindsor-va What actually happens?

Reply:
@stiehlrod It just returns empty
![Screenshot 2024-08-09 at 4 37 44 PM](https://github.com/user-attachments/assets/a917df70-5de8-430c-8fd6-3b64878f97d6)